### ISSUE TITLE
New project korge/ktor version issue

### ISSUE THREAD

**Issue by rafi0** [korlibs/korge#2211]
When i create a new project using korge/ktor template, the korge version there is 4.0.5a. I compiles and is fine.
When i update the version to something newer - 5.4.0 for example, during compilation i get this:

![Zrzut ekranu 2024-03-29 213754](https://github.com/korlibs/korge/assets/33195725/7f504578-1175-4e01-a5dd-bdcd604f83c2)

```
* What went wrong:
Error resolving plugin [id: 'com.soywiz.kproject', version: '0.2.7']
> The request for this plugi
```

**Comment by soywiz** [comment]
Yes. kproject is not exposed like that already, so that template is outdated. Eventually kproject will be replaced with amper. For now, here is a working settings.gradle.kts compatible with the latest version of korge that includes kproject:

https://github.com/korlibs/korge-hello-world/blob/main/settings.gradle.kts
